src/nvme/log.c

```diff
@@ -26,13 +26,18 @@
 #define LOG_CLOCK CLOCK_MONOTONIC
 #endif
 
-static nvme_root_t root;
+static struct nvme_log def_log = {
+	.fp = NULL,
```

Comment:
Or should we initialize this to stderr?

Reply:
If changed the `struct nvme_log` parameter: `FILE *fp` to `int fd` the `STDERR_FILENO` definition can be used instead. By the way the static data initialized by 0 so not necessary to initialize the parameters by `NULL` or `false` I think.

Reply:
Good idea with the filedescriptor. Let's do this, so we have sane defaults.

I'll keep the explicit initializing in place, I'd prefer to have all members listed so it's easy to read what the defaults are.